easyfsl/methods/utils.py

```diff
@@ -54,6 +54,19 @@ def k_nearest_neighbours(features: Tensor, k: int, p_norm: int = 2) -> Tensor:
     return distances.topk(k, largest=False).indices[:, 1:]
 
 
+def power_transform(features: Tensor, power_factor: float) -> Tensor:
```

Comment:
Do you really need a separate method to do this?
- it seems like it's used only once  -> no need for factorization
- it's quite simple: a one-liner self-explicit formula

Reply:
I think it can be a useful utils for users who'd want to use power-transform with other FSL methods